src/main/java/com/iexec/sms/tee/config/TeeWorkerInternalConfiguration.java

```diff
@@ -83,8 +83,7 @@ SconeServicesProperties sconeServicesProperties(
             final TeeAppProperties preComputeProperties,
             final TeeAppProperties postComputeProperties,
             @Value("${tee.scone.las-image}")
-            @NotBlank(message = "las image must be provided")
-            final String lasImage) {
+            @NotBlank(message = "las image must be provided") final String lasImage) {
```

Comment:
IntelliJ auto-save issue
```suggestion
            @NotBlank(message = "las image must be provided")
            final String lasImage) {
```